src/Slit.cc

```diff
@@ -1001,17 +1001,21 @@ void Slit::enterNotifyEvent(XCrossingEvent &ce) {
         toggleHidden();
     }
 
-    if (!m_timer.isTiming() && (m_rc_auto_hide && isHidden()) ||
-       (m_rc_auto_raise && m_layeritem->getLayerNum() != ResourceLayer::ABOVE_DOCK))
+    if (!m_timer.isTiming()
```

Comment:
So now we have the lines to spare? :P
Supposingly idempotent astyle-like changes need to be another commit by its own.
(And personally I'd never do that because you get a git-fence in return for nothing else)

Reply:
Sorry, this was my editor trying to enforce 80-character line limits. Also moot because of separation.